envtest/suite_test.go

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright 2019 IBM Corp.
```

Comment:
do we have a ticket to check which year to put here?

Reply:
sort of. anyway, I think we need the [current year for new files, and not change existing files](https://matija.suklje.name/how-and-why-to-properly-write-copyright-statements-in-your-code#how-to-write-a-good-copyright-statement-and-license-notice)

Reply:
so do you want to remove [update-copyright.sh](https://github.com/IBM/ibm-block-csi-operator/blob/develop/hack/update-copyright.sh) script?

Reply:
I'm meant not change existing years. I think this script only adds to files without years, so it might be better than nothing.
my point is, for this PR, we need new files to have the current year

Reply:
done